js/renovation/ui/grids/data_grid_light/views/footer.tsx

```diff
@@ -0,0 +1,25 @@
+import {
+  Component, JSXComponent, ComponentBindings,
+} from '@devextreme-generator/declarations';
+import { createPlaceholder } from '../../../../utils/plugin/context';
+import { Placeholder } from '../../../../utils/plugin/placeholder';
+
+export const FooterPlaceholder = createPlaceholder();
+
+export const viewFunction = (): JSX.Element => (
+  <Placeholder type={FooterPlaceholder} />
```

Comment:
Is it necessary to create separate widget for such simple JSX? May be create Placeholder inside DataGridLight.tsx?